pkg/services/secrets/kvstore/migrations/datasource_mig.go

```diff
@@ -33,9 +32,8 @@ func ProvideDataSourceMigrationService(
 ) *DataSourceSecretMigrationService {
```

Comment:
should we change the filename to match this? `ds_secret_mig.go`

Reply:
I thought since this is in the secrets/kvstore/migrations folder the "secret" part on the name of the file was a little redundant, so I kept it as datasource_mig to follow the same pattern of other migrations in Grafana, making it easier to find all migrations related to datasources.